libs/styles/iframe.css

```diff
@@ -28,6 +28,7 @@
 
 @media (max-width: 1200px) {
   .dialog-modal.commerce-frame .milo-iframe {
+    padding-bottom: 90vh;
```

Comment:
Wouldn't this get overridden by the next line?

Reply:
It's used as a fallback in case the value from the next line can not be applied, which seems to be the case for older iPads and other devices/browsers. At least that's my understanding

Reply:
Yes,That's correct.
The dvh unit is not supported by ipad 7th gen in my case and the fallback scenario has been tested in the same device.

Reply:
got it, thank you. It would be great to add a small comment for this use-case.

Reply:
I have added the comment.